spec/routemaster/integration/cache_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+require 'spec/support/uses_redis'
+require 'spec/support/uses_dotenv'
+require 'routemaster/cache'
+require 'webrick'
+
+RSpec.describe 'Requests with caching' do
```

Comment:
Nice — thanks for adding some integration testing. For extra karma, make the test logging a little less verbose?